### ISSUE TITLE
after login asking redirect to authorize page.

### ISSUE THREAD

**Issue by pratu103** [jublonet/codebird-js#129]
after login asking redirect to  https://api.twitter.com/oauth/authorize .there show PIN. then what to do cont understanding.
    $('#auth').click(function(e) {
cb.__call(
    "oauth_requestToken",
    {oauth_callback: "oob"},
    function (reply,rate,err) {
        if (err) {
            console.log("error response or timeout exceeded" + err.error);
        }
        if (reply) {
            // stores it
            cb.setToken(reply.oauth_token, reply.oauth_token_secret);
			aaa=reply;
            // gets the authorize screen URL
            cb.__call(
                "oauth_authorize",
                {},
                function (auth_url) {
				
                    window.codebird_auth = window.open(auth_url);
}
});
 
how to handle that PIN and next to continue. pls  Suggest me .

**Comment by jublo** [comment]
tl;dr:
Your application must allow the user to input this PIN to complete the flow. The PIN code must be passed as the value for oauth_verifier for the POST oauth/access_token request.

----

https://dev.twitter.com/oauth/reference/post/oauth/access_token

Hope this helps? :)
